app/vmselect/main.go

```diff
@@ -736,6 +736,7 @@ func proxyVMAlertRequests(w http.ResponseWriter, r *http.Request) {
 		// Forward other panics to the caller.
 		panic(err)
 	}()
+	r.URL.Path = strings.TrimPrefix(path, "prometheus")
```

Comment:
The path should not have `prometheus` prefix  as it was stripped before at https://github.com/VictoriaMetrics/VictoriaMetrics/blob/74b03c93a686921c9a68a150d002a459d67e77c5/app/vmselect/main.go#L109, right ?